.gitignore

```diff
@@ -1,4 +1,5 @@
 # Faasm
+dev/
```

Comment:
In the new docker-compose set-up we put all the development-related stuff in `dev`